string shop_id
---
pnp_msgs/ActionResult[] result
---
